frontend/src/pages/modelServing/screens/metrics/MetricsPage.tsx

```diff
@@ -34,16 +34,28 @@ const MetricsPage: React.FC<MetricsPageProps> = ({ title, breadcrumbItems, type,
       description={null}
       empty={false}
       headerAction={
-        tab === MetricsTabKeys.BIAS && (
-          <Button
-            isDisabled={statusState.type !== TrustyInstallState.INSTALLED}
-            variant="link"
-            icon={<CogIcon />}
-            onClick={() => navigate('../configure', { replace: true })}
-          >
-            Configure
-          </Button>
-        )
+        <Flex>
+          <FlexItem>
+            {tab === MetricsTabKeys.BIAS && (
+              <Button
+                isDisabled={statusState.type !== TrustyInstallState.INSTALLED}
+                variant="link"
+                icon={<CogIcon />}
+                onClick={() => navigate('../configure', { replace: true })}
```

Comment:
same as above, we need to pass a `state` with deployed model name and namespace right?